### DISCUSSION TITLE
Welcome to US-Zip-Codes-JSON Discussions!

### DISCUSSION THREAD

**Discussion by millbj92** [millbj92/US-Zip-Codes-JSON#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arnosolo** [comment]
Thanks you, sir. You save me a lot time.

**Reply by millbj92** [reply]
You're welcome! Glad you find it useful :)
